packages/neon_lints/lib/flutter.yaml

```diff
@@ -1,4 +1,5 @@
-include: package:neon_lints/dart.yaml
+include: package:neon_lints/flutter_package.yaml
```

Comment:
I think flutter_package should import flutter and not the other way around.

Reply:
I thought the strictest rules are used in a package and a non package can relax those rules.

Otherwise we'd have strict base -> relax for flutter non package -> tighten them again for flutter package

Reply:
I didn't think of flutter non package as relaxing the rules but it makes sense

Reply:
we can also base all rules on the base rule

Reply:
Hmm, but then we have duplicate stuff again. Let's just leave it like it is